pyproject.toml

```diff
@@ -1,4 +1,151 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
```

Comment:
```suggestion
```

- current version of setuptools (70.1+) does not need wheel at all
- older versions of setuptools would fetch wheel when building wheels (but not sdists)